qiita_db/handlers/processing_job.py

```diff
@@ -45,6 +50,30 @@ def _get_job(job_id):
     return job
 
 
+def _job_completer(job_id, payload):
+    """Completes a job
+
+    Parameters
+    ----------
+    job_id : str
+        The job to complete
+    payload : dict
+        The parameters of the HTTP POST request that is completing the job
+    """
+    payload_success = payload['success']
+    if payload_success:
+        artifacts = payload['artifacts']
+        error = None
+    else:
+        artifacts = None
+        error = payload['error']
+    job = qdb.processing_job.ProcessingJob(job_id)
+    try:
+        job.complete(payload_success, artifacts, error)
+    except Exception as e:
```

Comment:
Would it be worth to save the traceback itself? Or is this information presented to the user?